src/screens/Settings/index.tsx

```diff
@@ -72,6 +86,12 @@ const Settings: React.FC = () => {
         navigation.navigate("Home");
     };
 
+    const handleNotification = async () => {
+        const response = await updateUser({show_notifications: !isSwitchOn}, data.token);
+        if (response.status === 200)
+            setIsSwitchOn(!isSwitchOn);
```

Comment:
I personally think that this can cause a lack of feedback to the user (e.g the user clicks on button but the their internet connection/server is slow, so they won't see the switch changing for a while and it will probably make them click multiple times on the button). What do you think about? @tsrrodrigues

Reply:
I thought about that @alvesitalo, but for me it was worse to change the button without change the attribute on backend. Thinking about that i made this way. Do you think it is better to change the button independent of the response?

Reply:
My guess is to you may be able to change the switch state (to user's click before line 90) and then, if response isn't 200 revert the switch state (line 92).